feature-libs/cart/base/components/add-to-cart/add-to-cart.component.html

```diff
@@ -24,6 +24,7 @@
       [cxOutlet]="CartOutlets.ADD_TO_CART_PICKUP_OPTION"
       [(cxComponentRef)]="pickupOptionCompRef"
     ></ng-template>
+    <ng-content select="[extra-buttons]"></ng-content>
```

Comment:
[MEDIUM]

Nice idea to use ng-content with named selector.
IMHO we'd need to just make the name of the selector more precise. It should describe accurately the location of the ng-content in the host component's template. (but not descrbe what will be projected inside - it's a decision of the parent component)

What about renaming it to `[above-button]` or `[above-add-to-cart-button]`?

Reply:
[OUTDATED] We are not using this approach anymore. We use normal CMS mapping as we need to support only Quick Buy on a cart page.